scripts/redshift/transformers/__init__.py

```diff
@@ -0,0 +1,63 @@
+import os
+import typing
+from enum import Enum
+
+
+class TableName(Enum):
```

Comment:
Do you think there's a way to have a single, authoritative record of the tables somewhere? Just trying to minimize the number of things we need to remember to change if we update the schema.

Reply:
I'm on board with that. This record would have two requirements: 1) programmatically readable and 2) accessible from *. In code as an `Enum` satisfies (1), and as far as I can tell, * is the loader/notifications script. Do you think a globally accessible Enum in the codebase works? Or were you thinking the source of truth would be out of the code somewhere?